daemon/cmd/daemon.go

```diff
@@ -936,6 +934,13 @@ func NewDaemon(ctx context.Context, cleaner *daemonCleanup,
 		case option.Config.EgressMasqueradeInterfaces != "":
 			msg = fmt.Sprintf("BPF masquerade does not allow to specify devices via --%s (use --%s instead).",
 				option.EgressMasqueradeInterfaces, option.Devices)
+		case option.Config.TunnelingEnabled():
+			if !hasFullHostReachableServices() {
```

Comment:
```suggestion
		case option.Config.TunnelingEnabled() && !hasFullHostReachableServices() {
```

Reply:
the  suggestion result in error `daemon/cmd/daemon.go|938 col 35| expected ':', found '{' (and 3 more errors)`, what about 
```
case option.Config.TunnelingEnabled() && !hasFullHostReachableServices():
                        msg = "BPF masquerade requires full host reachable services enabled in tunnel mode."
```

Reply:
Yep, that's what I meant to write :+1: